### ISSUE TITLE
[$40][MSFT-268] [Registration] NVDA : NVDA announces "clickable" and "blank" at end of the page

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2912]
**Steps to Reproduce**
1. Launch NVDA
2. Launch "https://accounts.topcoder.com/member/registration" in Chrome
3. Navigate using virtual cursor (arrow keys)
4. Navigate to end of the page
5. After the announcement of "All rights reserved" press down arrow key

**Expected Result**
After the announcement of ""All rights reserved" if down arrow is pressed, again and again ""All rights reserved" only to be announced

**Actual Result**
Note that NVDA announces "Clickable" and "Blank"

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FNVDA%20announces%20clickable%20and%20blank%20at%20end%20of%20the%20page

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096243 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by benoitrion** [comment]
@cwdcwd Hi, how can I access the screenshot inside the sharepoint ? 
It seems that my outlook account needs to be added.

**Comment by benoitrion** [comment]
@cwdcwd @veshu Is the registration page code inside community-app ? I can't find anything inside the code base.

**Comment by veshu** [comment]
@benoitrion Please check https://apps.topcoder.com/forums/?module=Thread&threadID=940852&start=0

**Comment by veshu** [comment]
@benoitrion login and registration related code is at https://github.com/appirio-tech/accounts-app

Challenge listing/detail and submission page related code is in this repository.

cc @mishacucicea

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096243 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096243 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by mfikria** [comment]
@veshu are all these issues open for public?

**Comment by veshu** [comment]
@mfikria Yes, for all topcoder member.

**Comment by mfikria** [comment]
@veshu could you invite me to the repo so i could self-assigned the issue?

**Comment by veshu** [comment]
@mfikria Please check here for challenge detail and other info https://github.com/topcoder-platform/community-app/issues/2915 also check the forum for other info.

**Comment by mfikria** [comment]
@veshu alright thanks!

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096243 has been updated - it has been assigned to mfikria.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
@tc-mfikria, please sign-up with Topcoder X tool<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096714 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096714 has been updated - it has been assigned to mfikria.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tc-mfikria** [comment]
@veshu i've tried fixed and opened a PR in https://github.com/appirio-tech/accounts-app/pull/262

**Comment by drasticdpk** [comment]
Verified on Windows/Chrome/NVDA.

**Comment by drasticdpk** [comment]
@nkumar-topcoder @Oanh-and-only-Oanh ,  we are still listening "clickable" and "blank", please confirm.
here is the reference :
https://monosnap.com/file/GGDqPS3YTUvaWHeLk1PXgEMZaSD4rW

**Comment by nkumar-topcoder** [comment]
> @nkumar-topcoder @Oanh-and-only-Oanh , we are still listening "clickable" and "blank", please confirm.
> here is the reference :
> https://monosnap.com/file/GGDqPS3YTUvaWHeLk1PXgEMZaSD4rW

@drasticdpk its same fix which was verified in qa. Can you refresh or use incognito please ?

**Comment by crazyk07** [comment]
@nkumar-topcoder  I'm getting the same thing as @drasticdpk  here, using incognito on chrome

**Comment by nithyaasworld** [comment]
Its not working for me too.

But it worked for me during the smoke test today. So its most likely deployment issue.

**Comment by nkumar-topcoder** [comment]
> Its not working for me too.
> 
> But it worked for me during the smoke test today. So its most likely deployment issue.

@nithyaasworld i am surprised, this is the PR 2912 and it's in master branch. 
https://github.com/appirio-tech/accounts-app/pull/262/files

**Comment by nithyaasworld** [comment]
Thanks @nkumar-topcoder for checking! I think the developer @tc-mfikria will be able to give us more info on why its failing in Production environment.

**Comment by nithyaasworld** [comment]
It passed quality check by 3 different persons (@mishacucicea @drasticdpk and me) before deploying to production. So just guessing that there is somewhere something we are missing in the Production environment.

**Comment by tc-mfikria** [comment]
@nithyaasworld @nkumar-topcoder maybe it is because the browser version used is different

**Comment by nkumar-topcoder** [comment]
@tc-mfikria , can you pls verify in dev and prod and get back please.

**Comment by tc-mfikria** [comment]
Sorry, I cannot fix the issue, release to open for pickup

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096714 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by mishacucicea** [comment]
Changes will be reverted on this one. Putting on hold till we figure out the `chameleon` issue.

**Comment by r0hit-gupta** [comment]
@mishacucicea it's probably fixed and ready for Prod QA.

**Comment by nkumar-topcoder** [comment]
> @mishacucicea it's probably fixed and ready for Prod QA.

@r0hit-gupta can you please verify both PROD and DEV branch changes for this fix. If not available. Please take the file from `DEV` branch, apply the fix, test locally and PR for `DEV` branch. We have issues on deploy/testing with `qa-accessibility` hence.

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder @mishacucicea this issue is solved and ready for Prod QA. The PR https://github.com/appirio-tech/accounts-app/pull/295 simply reverts the unnecessary changes.

**Comment by nkumar-topcoder** [comment]
@mishacucicea @r0hit-gupta @tosha5252 @drasticdpk this is merged and `ready for qa` status. again this is merged to `qa-accessibility` if fix not reflected let me know.

**Comment by tosha5252** [comment]
Now there is no announcement at all of 'All Rights Reserved'...NVDA just starts back at the beginning of the page

Win10/Chrome/NVDA

**Comment by crazyk07** [comment]
@tosha5252  Is the expected behavior for the screenreader suppose to announce "All right reservered"?  I don't think it's dynamic content at all, so I don't think it should have been read in the first place

**Comment by tosha5252** [comment]
@crazyk07 

Screen reader does not announce "Clickable" and "Blank" or "All rights reserved"

"All rights reserved" is not dynamic content, so it is not necessary for the screen reader to read it.

Win10/Chrome/NVDA

**Comment by tosha5252** [comment]
Reverified and is working correctly 

Win10/Chrome/NVDA

@nkumar-topcoder

**Comment by tosha5252** [comment]
working correctly

Win10/Chrome/NVDA

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096714<br/><br/>```This is an automated message for crazyk via Topcoder X```
